proto/process/agent.proto

```diff
@@ -1591,3 +1601,106 @@ message VPACondition {
 	string Reason = 4;
 	string Message = 5;
 }
+
+// https://github.com/kubernetes/api/blob/master/autoscaling/v2/types.go
```

Comment:
let's don't use `master` as it keeps changing, we need to find the version we will use in agent